colossalai/inference/config.py

```diff
@@ -43,8 +45,24 @@ class InferenceConfig:
     max_seq_len: Optional[int] = None
     quant_mode: Optional[str] = None
     revision: Optional[str] = None
-    # TODO: beam search is not support for now
     beam_width: int = 1
+    # TODO: beam search is not support for now
+    ratio: Optional[float] = 1.2
```

Comment:
2 actions need for this arg `ratio`:
1. name it clearly, generally in the form of `xxx_ratio`
2. add its doc to the docstring.

Reply:
done